srcpkgs/uucp/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'uucp'
+pkgname=uucp
+version=1.07
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr --mandir=/usr/share/man --with-newconfigdir=/etc/uucp --sbindir=/usr/bin"
+makedepends="wget"
+short_desc="Taylor UUCP is a free implementation of UUCP"
+maintainer="MarcoAPC <marcoaureliopc@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://www.gnu.org/software/uucp/uucp.html"
+distfiles="${GNU_SITE}/$pkgname/$pkgname-$version.tar.gz"
+checksum=060c15bfba6cfd1171ad81f782789032113e199a5aded8f8e0c1c5bd1385b62c
+patch_args="-Np1"
+
+system_accounts="uucp"
```

Comment:
Should be prefixed with underscore per Manual.md.

Reply:
Done.

Reply:
Uses of the account name should be changed too.

Reply:
Done.